rfqmsg/buy_accept.go

```diff
@@ -15,13 +15,23 @@ import (
 const (
 	// Buy accept message type field TLV types.
 
-	TypeBuyAcceptID        tlv.Type = 0
-	TypeBuyAcceptAskPrice  tlv.Type = 2
-	TypeBuyAcceptExpiry    tlv.Type = 4
-	TypeBuyAcceptSignature tlv.Type = 6
-	TypeBuyAcceptAssetID   tlv.Type = 8
+	TypeBuyAcceptVersion   tlv.Type = 0
+	TypeBuyAcceptID        tlv.Type = 2
+	TypeBuyAcceptAskPrice  tlv.Type = 4
+	TypeBuyAcceptExpiry    tlv.Type = 6
+	TypeBuyAcceptSignature tlv.Type = 8
+	TypeBuyAcceptAssetID   tlv.Type = 10
```

Comment:
Can we consolidate the buy/sell messages with what we had outlined here? https://github.com/Roasbeef/blips/pull/3

Reply:
I'll spin that out into another PR. I think this PR is ready to be merged.